start.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+shopt -s expand_aliases
+install_flag='false'
+production_flag='false'
+
+python3_cmd=`which python3`
+python_cmd=`which python`
+node_cmd=`which node`
+npm_cmd=`which npm`
+
+BACKEND_PORT=5000
+BACKEND_HOST="0.0.0.0"
+FRONTEND_PORT=4200
+
+
+if [ -z "$node_cmd" ]; then
+    echo "Node not found. Cannot continue"
+    exit 1
+fi
+
+if [ -z "$npm_cmd" ]; then
+    echo "Npm not found. Cannot continue"
+    exit 1
+fi
+
+
+if [ -n "$python3_cmd" ]; then
```

Comment:
On my system this won't work, since `python3==3.8.*`. Could we change that that we just `export` our desired python version beforehand? And maybe you perform the `which` commands only when the `env variable` is not set?

Reply:
I updated to include a specific check for a `python` environment variable